README.md

```diff
@@ -64,6 +97,19 @@ spec:
 EOS
 ```
 
+
+## Key Features
+
+* Binding with backing services represented by Kubernetes resources including third-party CRD-backed resources.
+* Binding secret generation based on OLM descriptors and CRD/CR annotations.
```

Comment:
Since both OLM descriptors and annotations provide the same functionality, why do we need to support both?

Reply:
OLM descriptors give us both the same semantics annotations give us to extract binding values from resources, with the added benefit of providing hints for OpenShift's UI to enrich the user's experience.

Reply:
I believe each feature should have its own bullet:

```md
* Extract binding values based on annotations present in CRD/CR
* Extract binding values based on OLM descriptors 
```

Reply:
Done!